src/pages/Contact/Contact.jsx

```diff
@@ -0,0 +1,40 @@
+import Layout from "../../components/Layout/Layout";
+import LabelledInputTextField from "../../components/labelledInputTextField/labelledInputTextField";
+import Button from "../../components/button/Button";
+import "./contact.scss";
+
+const Contact = () => {
+  return (
+    <>
+      <Layout>
+        <main className="contact-container">
```

Comment:
Using BEM in react, the parent class should be the name of the component. so this should be "contact" and everything inside it should be "contact__"